server/controller/common/getBlogs.js

```diff
@@ -5,7 +5,7 @@ const getAllBlogs = async (req, res, next) => {
   try {
     const { rows } = await getBlogs();
     if (!rows) {
-      throw boomify(500, 'Internal Server Error');
+      throw boomify(203, 'There is no blogs');
```

Comment:
i think 203 status is not suitable for your case as it means 203 means **`Non-Authoritative Information (or, "The server successfully processed the request, but is returning information that may be from another source")`**
so you should either use **`204 no content`** as Mohammed mentioned  [here](https://github.com/GSG-G9/beauty-salon/pull/59#discussion_r605916212) or **`200 status`** if you want send a message "no blogs"